static/javascript/launch.js

```diff
@@ -1,4 +1,13 @@
 // uuidv4: from https://github.com/kelektiv/node-uuid
+function escapeHtml(unsafe) {
```

Comment:
Is this actually used anywhere? How come this has been added?

Reply:
Oops forgot to apply the escapeHTML function. Added now.

Reply:
It was a suggested change so "any text taken from the DOM and used to set innerHTML is properly escaped to prevent XSS attacks".